addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -95,7 +95,7 @@ private _id = switch (_type) do {
     };
     case "turret": {
         if (_applyRetroactively) then {
-            [GVAR(oldUnit), GVAR(oldUnit) call CBA_fnc_turretPath, []] call _function;
+            [GVAR(oldUnit), GVAR(oldTurret), []] call _function;
```

Comment:
Same. Why?

Reply:
we're already using `GVAR(oldUnit)`, why not do the same for other vars

Reply:
All other besides player calculate the current value.

Reply:
If all the other ones drove off a cliff, should this one as well?